src/constants.F90

```diff
@@ -84,6 +84,7 @@ module constants
        ZERO             = 0.0_8,             &
        HALF             = 0.5_8,             &
        ONE              = 1.0_8,             &
+       SQRT_TWO         = 1.4142135623731_8, &
```

Comment:
This loses precision -- just do `SQRT_TWO = sqrt(2.0_8)`. Also, doesn't look like this is used?